fppanalysis/conditional_averaging.py

```diff
@@ -41,7 +44,8 @@ def None
 
     places = np.where(sgnl > smin)[0]
     assert len(places) > 0, "No conditional events"
-    print("places to check:{}".format(len(places)), flush=True)
+    if print_verbose == True:
```

Comment:
You can simply write `if print_verbose:` instead of `if print_verbose == True`. This makes the code concise.